<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QDialog" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>131</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Flop: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Trail: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>MechTrail:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_trail">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_mTrail">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_flop">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>140</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>HT angle</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>72.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Rake</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_2">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="value">
       <number>35</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>dia</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>wide</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="value">
       <number>23</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>559</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>622</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>149</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Results:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
